utils/udpredirect/udpredirect.cpp

```diff
@@ -17,7 +17,7 @@ int main(int argc, char *argv[]) {
     if (argc!=3 && argc!=5) {
         printf("Usage: %s our-ip our-port send-to-ip send-to-port\n",argv[0]);
         printf("Usage: %s our-ip our-port             # echo mode\n",argv[0]);
-        exit(1);
+        return 1;
```

Comment:
Are the only return possibilities 0 or 1? If not, using constants to codify the return codes might be a good idea.

Reply:
yes, now using EXIT_SUCCESS and EXIT_FAILURE